packages/contracts/src/contracts/current/protocol/Exchange/MixinAssetProxyDispatcher.sol

```diff
@@ -66,11 +66,19 @@ contract MixinAssetProxyDispatcher is
         external
         onlyOwner
     {
```

Comment:
The doc comment `however, an asset proxy may be registered to multiple ids.` Is this still possible? Since we've hardcoded the assetProxyID in the assetProxy and only expose a single `getProxyId()` function.

Reply:
You're right, will remove this.